src/components/Tag.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import { css } from '@emotion/core';
+import { Link } from 'gatsby';
+import styled from '@emotion/styled';
+import { rgba } from 'polished';
+
+const isInteractive = ({ as, interactive }) =>
+  interactive || as === 'a' || as === Link;
+
+const Tag = styled.span`
```

Comment:
TODO: Need to update to use the gatsby theme tag component once a new version is released.